docs/3_project_life_cycle_session.md

```diff
@@ -61,8 +61,7 @@ A project can be deleted in cases of incorrect project information or errors, an
 
 Projects are **Available** for a certain (chosen by the units) period of time, Days in Available (DiA). After that their status automatically changes to **Expired**. Data cannot be changed in this state, but the project can be re-released up to *two times*. After two re-releases, a project can only be archived.
 
-ToDO add task for checking project status history and note deadline
-- [ ] TASK: 
+- [ ] TASK: Check the current project status and the status history. Observe the deadline date for the current status.
```

Comment:
_Observe the deadline date for the current status._

This part is slightly confusing to me - but does it mean the deadline date for whatever the current status is (but isn't it always In Progress at this point if users are following the steps)?

Reply:
At that point, the project should be expired, no?

Reply:
Cool, I understand.

Reply:
> At that point, the project should be expired, no?

No, how could they expire so soon

Reply:
my bad. I got confused and thought there was a task about expiring a project